sdk/communication/communication-call-automation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/communication-call-automation",
-  "version": "1.3.0-beta.1",
+  "version": "1.4.0-beta.2",
```

Comment:
update is also needed for the version string in src/models/constants.ts.  We have an automation from azsdk-bot to increment version in files other than package.json and CHANGELOG.md for a package after its release, but you need to provide some information in package.json, similar to https://github.com/Azure/azure-sdk-for-js/blob/1d3c61badc8ecaa5e6c404862fb0d8c98b784259/sdk/appconfiguration/app-configuration/package.json#L57-L72

Reply:
Is this something newly added? just wondering why we are not seeing it in the main branch. And also we usually release  from feature branch and then manually we merge it main.

Reply:
The auto incrementing of versions? It has been around for a while, and I think it also works for feature branch.